README.md

```diff
@@ -1,114 +1,51 @@
-Prebid Server forms OpenRTB requests, sends them to the appropriate adapters, concatenates the responses, and returns them
-to the client.
-
-A few key points:
- * No ranking or decisioning is performed by Prebid Server. It just proxies requests.
- * No ad quality management (malware, viruses, deceptive creatives) is performed by Prebid Server
- * Prebid Server does no fraud scanning and does nothing to prevent bad traffic.
-
-# User synching
-Prebid Server provides a `/setuid` endpoint that allows adapters to push in their user IDs. These are stored in a cookie named,
-creatively, `uids`. To see stored cookies, call `/getuids`. To set an optout cookie, call `/optout`. When an adapter doesn't
-have a synched cookie, a `no_cookie` response is returned with a usersync URL that the client should call via asynchronous pixel
-or equivalent. If Prebid Server doesn't have a cookie set, a preemptive `no_cookie` response is returned to allow the client
-to ask for user consent and drop a cookie.
-
-# Logging
-Prebid Server does no server-side logging. It can stream metrics to an InfluxDB endpoint, which are aggregated as a time series.
-Prebid Server has no user profiling or user-data collection capabilities.
-
-# Usage
-## Without Docker
-### Prerequisites
-* [Go](https://www.golang.org)
-* [Glide](https://glide.sh/)
-
-# Hosted version
-AppNexus is hosting a version (generally bleeding-edge of this repo including some in-flight pull requests) at https://prebid.adnxs.com.
-
-### Getting
-1. Install glide: https://github.com/Masterminds/glide#install
-2. `cd $GOPATH`
-3. `git clone https://github.com/prebid/prebid-server src/github.com/prebid/prebid-server`
-4. `cd src/github.com/prebid/prebid-server`
-5. `glide install`
-6. `./validate.sh`
-
-### Running
-To compile a binary and run locally:
-```
-$ make build
-$ ./prebid-server -v 1 -logtostderr
-```
+# Prebid Server
 
-## With Docker
-### Prerequisites
-* [Docker](https://www.docker.com)
+Prebid Server is an open source implementation of Server-Side Header Bidding.
+It is managed by [Prebid.org](http://prebid.org/overview/what-is-prebid-org.html),
+and upholds the principles from the [Prebid Code of Conduct](http://prebid.org/wrapper_code_of_conduct.html).
 
-### Compiling an alpine binary
-The Dockerfile for prebid-server copies the binary in the root directory to the
-docker container, and must be specifically be compiled for the target
-architecture (alpine).
+For more information, see:
 
-```
-$ docker run --rm -v "$PWD":/go/src/github.com/prebid/prebid-server \
--w /go/src/github.com/prebid/prebid-server \
-billyteves/alpine-golang-glide:1.2.0 \
-/bin/bash -c 'glide install; go build -v'
-```
+- [A Beginner's Guide to Header Bidding](http://adprofs.co/beginners-guide-to-header-bidding/)
+- [Server-side Header Bidding Explained](http://www.adopsinsider.com/header-bidding/server-side-header-bidding/)
```

Comment:
Any reason in particular why we're including these two links?

Reply:
Yes... no... maybe? This is what I was thinking:

GitHub READMEs usually start with a "what is this project and why might I want it?" section. In this case, it's complicated... so I linked to two articles that I thought were well written and hit the key points.

The first link argues for traditional header bidding over traditional ad servers.
The second link describes some problems with traditional header bidding and described how moving it server-side may help. I also liked how it was honest about the costs & benefits (improved performance, but lower cookie sync rates).

Not sure if there are better articles out there for these purposes, though. I just googled and grabbed the first few which looked good to me.